packages/location-field/src/index.js

```diff
@@ -430,6 +430,9 @@ class LocationField extends Component {
         .filter(feature => feature.properties.source !== "transit")
         .slice(0, suggestionCount);
 
+      const nonOtherFeaturesPresent =
```

Comment:
Rename to `standardFeatures`?

Reply:
This is much better thank you! Renamed in 026fb0c